code/utils/path_bold.py

```diff
@@ -17,7 +17,7 @@ def path_bold(subject, task_num):
 		this is the path of the bold.
 	"""
 
-	sub_path = os.path.realpath(subject)
-	bold_path = sub_path + '/BOLD' + '/' + task_num
+	sub_path = os.path.realpath('preprocessed_ds105/' + subject) 
```

Comment:
added preporcessed_ds105 but should be able to find subjects/fmri data regardless of pwd